packages/fether-react/src/Accounts/CreateAccount/AccountImportOptions/AccountImportOptions.js

```diff
@@ -67,6 +90,22 @@ class AccountImportOptions extends Component {
     }
   };
 
+  hasExistingAddressForImport = addressForImport => {
+    const { accountsInfo } = this.props;
+    const isExistingAddress = Object.keys(accountsInfo)
+      .map(address => address && address.toLowerCase())
+      .includes(addressForImport);
+
+    if (isExistingAddress) {
+      this.setState({
+        isLoading: false,
+        error: `Account already loaded. Address ${addressForImport} is already in the account list`
```

Comment:
I would go for something much shorter along the line:
"Account 0x1234..1234 already listed!"
You don't have to see the whole address here.